engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/db2_engine_7.22_to_7.23.sql

```diff
@@ -0,0 +1,19 @@
+--
+-- Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+-- under one or more contributor license agreements. See the NOTICE file
+-- distributed with this work for additional information regarding copyright
+-- ownership. Camunda licenses this file to you under the Apache License,
+-- Version 2.0; you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+insert into ACT_GE_SCHEMA_LOG
+values ('1200', CURRENT_TIMESTAMP, '7.23.0');
```

Comment:
❓ **Open point**: Verify the correctness of value `1200`

**Hint**: The pattern is to increment the value by `100` every new minor release. 

The last value used is `1100` (see [example](https://github.com/camunda/camunda-bpm-platform/blob/9ff52fe0f51ba74416a577a635cba62278d283a4/engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/db2_engine_7.21_to_7.22.sql#L19))